curriculum/challenges/english/07-scientific-computing-with-python/learn-special-methods-by-building-a-vector-space/65fc8707f3af570332499991.md

```diff
@@ -0,0 +1,75 @@
+---
+id: 65fc8707f3af570332499991
+title: Step 50
+challengeType: 20
+dashedName: step-50
+---
+
+# --description--
+
+Still inside the `if` body, return a new instance of the current class by using `__class__` and unpacking `kwargs` with the `**` operator. This will be the vector resulting from the scalar multiplication.
```

Comment:
```suggestion
Still, inside the `if` body, return a new instance of the current class by using `__class__` and unpacking `kwargs` with the `**` operator. This will be the vector resulting from the scalar multiplication.
```